source/index.html.slim

```diff
@@ -13,6 +13,19 @@ section
       ' With a mixture of local and international speakers, there will be one day of workshops and one single-track day of talks. We'll cover topics that interest, excite, and delight Rubyists and the Ruby-curious of all levels.
     p
       ' It's being held on Thursday 2 November (workshops) and Friday 3 November (conference) at Te Papa on the Wellington waterfront.
+
+  .body-inner-wrapper__section#signup
```

Comment:
Why is this being added back in - due to merge conflicts or do you feel strongly about these sections?